lib/bundler/plugin.rb

```diff
@@ -45,6 +45,22 @@ def install(names, options)
       Bundler.ui.error "Failed to install plugin #{name}: #{e.message}\n  #{e.backtrace.join("\n ")}"
     end
 
+    # List installed plugins and commands
+    #
+    def list
+      output = ""
```

Comment:
I just droped Bundler.ui.info in the end with the output information. Not sure if is the best.